sdk/textanalytics/Azure.AI.TextAnalytics/src/Internal/CustomEntitiesTaskParameters.cs

```diff
@@ -5,8 +5,9 @@
 
 namespace Azure.AI.TextAnalytics.Models
 {
+    /// <summary>
+    /// EntitiesTaskParameters class.
```

Comment:
I think the summary is not needed as this is an internal class. If you still want to modify it, `Custom` is needed

Reply:
I had them there to be consistent with the extract summary internal models. I've adjusted the docstring to include `Custom `, but I can remove them if you think that would be better.